Modules/TOF/src/TaskRaw.cxx

```diff
@@ -47,9 +47,23 @@ void RawDataDecoder::rdhHandler(const o2::header::RAWDataHeader* rdh)
 {
   mCounterRDH[rdh->feeId & 0xFF].Count(0);
 
+  //Fatal
   if ((rdh->detectorField & 0x00010000) != 0) {
     mCounterRDH[rdh->feeId & 0xFF].Count(1);
   }
+
+  //Trigger expected and served
+  if (rdh->stop) { // if RDH close
+    int triggerserved = ((rdh->detectorField & 0xFF00070F) >> 24);
+    int triggerreceived = ((rdh->detectorField & 0x00FF070F) >> 16);
+    if (triggerserved < triggerreceived) {
+      //Trigger error
+      mCounterRDH[rdh->feeId & 0xFF].Count(2);
+    }
+    //Trigger efficiency
```

Comment:
This is not strictly the trigger efficiency, maybe you can consider adding: `Ingredients for the trigger efficiency` (also for the future)